src/main/java/com/marginallyclever/makelangelo/Makelangelo.java

```diff
@@ -876,6 +877,8 @@ private void onChangeSlider(ChangeEvent e) {
         myTurtleRenderer.setLast(top);
         labelRangeMin.setText(Integer.toString(bottom));
         labelRangeMax.setText(Integer.toString(top));
+		SaveGCode.setTrimHead(bottom);
```

Comment:
Settings static values in the a bad pattern. Every time the user ask to save the gcode, a new `SaveGCode` is instanciated, so thoses values can be set in the constructor of even better in the `save()` method